TypeScriptDefinitions/async.d.ts

```diff
@@ -0,0 +1,165 @@
+// Type definitions for Async 1.4.2
```

Comment:
What is this for?  Part of the `async` package?  Does it need to be in this location?